src/main/java/frc/team4909/robot/subsystems/elevator/commands/SetElevatorPosition.java

```diff
@@ -1,19 +1,28 @@
 package frc.team4909.robot.subsystems.elevator.commands;
 
+import edu.wpi.first.wpilibj.command.Command;
 import edu.wpi.first.wpilibj.command.InstantCommand;
 import frc.team4909.robot.subsystems.elevator.ElevatorSubsystem;
 import frc.team4909.robot.Robot;
 
 public class SetElevatorPosition extends InstantCommand {
-    private int setpoint;
+    private int setpoint, threshold;
 
-    public SetElevatorPosition(int setpoint) {
-        this.setpoint = setpoint;       
+    public SetElevatorPosition(int setpoint, int threshold) {
+        this.setpoint = setpoint;  
+        this.threshold = threshold;     
         requires(Robot.elevatorSubsystem);
     }
 
     @Override
     public void initialize() {
-        Robot.elevatorSubsystem.holdingPosition = setpoint;
+        Robot.elevatorSubsystem.setPosition(setpoint);
+
+    }
+
+    @Override
+    protected boolean isFinished() {
```

Comment:
this *may* fix the instant command thing but it's very hacky, might as well use the proper class from the start

Reply:
^^ there should be an isfinished condition so it can be strung together in a semiauto